lib/exception_hunter/devise.rb

```diff
@@ -1,6 +1,8 @@
 module ExceptionHunter
   module Devise
     class SessionsController < ::Devise::SessionsController
+      skip_before_action :verify_authenticity_token
```

Comment:
isn't this automatically done on session controller?

Reply:
I don't think so, and it only causes problems on dev